rpms/s3/s3rpm.spec

```diff
@@ -336,14 +355,7 @@ rm -rf %{buildroot}
 %exclude /opt/seagate/cortx/s3/reset/precheck.pyo
 
 %post
-[ -f /opt/seagate/cortx/s3/conf/s3config.yaml ] || 
-    cp /opt/seagate/cortx/s3/conf/s3config.yaml.sample /opt/seagate/cortx/s3/conf/s3config.yaml
-[ -f /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml ] ||
-    cp /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml.sample /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml
-[ -f /opt/seagate/cortx/auth/resources/authserver.properties ] ||
-    cp /opt/seagate/cortx/auth/resources/authserver.properties.sample /opt/seagate/cortx/auth/resources/authserver.properties
-[ -f /opt/seagate/cortx/auth/resources/keystore.properties ] ||
-    cp /opt/seagate/cortx/auth/resources/keystore.properties.sample /opt/seagate/cortx/auth/resources/keystore.properties
+sh /opt/seagate/cortx/auth/scripts/swupdate/merge.sh
```

Comment:
we dont check return value of sh /opt/seagate/cortx/auth/scripts/swupdate/merge.sh, what if it fails to merge, in that case its should go back to old version

Reply:
Rollback cas not yet handled.